File: backend/SNS.Application/Projects/Contracts/ProjectParticipantDetailsDto.cs
namespace SNS.Application.Projects.Contracts;

public record ProjectParticipantDetailsDto(
    Guid ProfileId,
    string? ProfileImageUrl,
    string DisplayName,
    string? Specialization,
    int FollowersCount,
    int FollowingCount,
    bool IsFollowedByCurrentUser,
    string Role
);

File: SNS.Web/src/app/projects/contracts/project-participant-details.dto.ts
export interface ProjectParticipantDetailsDto {
    profileId: string;
    profileImageUrl: string | null;
    displayName: string;
    specialization: string | null;
    followersCount: number;
    followingCount: number;
    isFollowedByCurrentUser: boolean;
    role: string;
}
